lib/qbwc/response/item_site_query_rs.rb

```diff
@@ -0,0 +1,86 @@
+module QBWC
+  module Response
+    class ItemSitesQueryRs
+      attr_reader :records
+
+      def initialize(records)
+        @records = records
+      end
+
+      def handle_error(errors, config)
+        errors.each do |error|
+          Persistence::Object.handle_error(config,
+                                           error.merge(context: 'Querying item site query'),
+                                           'inventories',
+                                           error[:request_id])
+        end
+      end
+
+      def process(config = {})
+        return if records.empty?
+        puts 'ItemSitesQueryRs#process'
+
+        receive_configs = config[:receive] || []
+        inventory_params = receive_configs.find { |c| c['inventories'] }
+
+
+        if inventory_params
+          payload = { inventories: inventories_to_flowlink }
+          config = { origin: 'quickbooks' }.merge config.reject{|k,v| k == :origin || k == "origin"}
+
+          poll_persistence = Persistence::Polling.new(config, payload)
+          poll_persistence.save_for_polling
+
+          inventory_params['inventories']['quickbooks_force_config'] = 'true'
+          params = inventory_params['inventories']
+
+          Persistence::Settings.new(params.with_indifferent_access).setup
+        end
+
+        nil
+      end
+
+      private
+
+      def inventories_to_flowlink
+        records.map do |record|
+          {
+            list_id: record['ListID'],
```

Comment:
We also NEED an `id` field here for FlowLink. For normal products, it would be the name (`id: full_name(record)`), but I'm not exactly sure **what** this object is in QBE?

Does the `record['ListID']` refer to this same item every single time??
If not, I think maybe `id: full_name(record) + record['InventorySiteRef']['FullName']` would be the right thing. The item name + the inventory site is unique.

Reply:
Would full_name be the sku in Systum in that case?

Reply:
The list id refers to the same item I believe but I think it would be better to use your idea of combining the full name and inventory site. It would make the id more readable

Reply:
FullName _might_ be the SKU in Systum. It also might be something more. FullName sometimes could be Number1ParentName:AnotherParentName:SKU in QBE where the last part matches the SKU in Systum.
Thank god we haven't dealt with anything that crazy yet...